### PR TITLE
fix: change appnavigation to appnavigationcaption for the email account

### LINKED ISSUE #9756: Use NcNavigationCaption for account navigation element. Remove color circle

### PR DESCRIPTION
**Pull request by GretaD**
before
Screenshot from 2024-06-07 12-37-39
after
Screenshot from 2024-06-07 12-38-03
fixes #9756

### CODE REVIEW

**Comment by GretaD** [comment]
cc @marcoambrosini thats what you meant during the call, right?

File: src/components/NavigationAccount.vue [hunk 1]
```diff
@@ -21,12 +21,11 @@
   -->
 
 <template>
-	<AppNavigationItem v-if="visible"
+	<NcAppNavigationCaption v-if="visible"
 		:id="id"
 		:key="id"
 		:menu-open.sync="menuOpen"
```
Comment (ChristophWurst) [inline src/components/NavigationAccount.vue:27]: Does this prop exist? https://nextcloud-vue-components.netlify.app/#/Components/App%20containers/NcAppNavigation?id=ncappnavigationcaption

File: src/components/NavigationAccount.vue [hunk 2]
```diff
@@ -21,12 +21,11 @@
...
 		:name="account.emailAddress"
-		:to="firstMailboxRoute"
 		:exact="true"
```
Comment (ChristophWurst) [inline src/components/NavigationAccount.vue:29]: Same. This is not a router element

**Review by ChristophWurst — COMMENTED** [review]
:broom:

File: src/components/NavigationAccount.vue [hunk 3]
```diff
@@ -21,12 +21,11 @@
...
 		@update:menuOpen="onMenuToggle">
 		<template #icon>
```
Comment (marcoambrosini) [inline src/components/NavigationAccount.vue:31]: Also the bulletcolor component goes away

**Review by marcoambrosini — COMMENTED** [review]
more 🧹

File: src/components/NavigationAccount.vue (hunk 3 above)
Comment (marcoambrosini) [reply src/components/NavigationAccount.vue:31]: Sorry wrong line, I mean in `template #icon`

**Review by ChristophWurst — APPROVED** [review]

File: src/components/NavigationAccount.vue [hunk 4]
```diff
@@ -106,16 +99,15 @@
 		<template #extra>
```
Comment (GretaD) [inline src/components/NavigationAccount.vue:99]: NcAppNavigationCaption doesnt have an extra slot, need to find a way how to make account settings work

Comment (GretaD) [reply src/components/NavigationAccount.vue:99]: @marcoambrosini should i add an extra slot at appnavigataioncaption, or should we use a different component?

Comment (marcoambrosini) [reply src/components/NavigationAccount.vue:99]: Maybe use a <Fragment> component @GretaD https://www.npmjs.com/package/vue-fragment
cc @ShGKme @susnux for opinions

Comment (ShGKme) [reply src/components/NavigationAccount.vue:99]: Yes, fragment will work here for Vue 2. But we are usually using `vue-frag`: https://www.npmjs.com/package/vue-frag 

Mail already has it installed.

**Review by ShGKme — APPROVED** [review]
Tested, all fine.

Now it is not possible to open the inbox by clicking on the account name. I guess it is intended, because we cannot collapse it anyway and the inbox  link is always visible.

**Comment by ShGKme** [comment]
(auto-merged after restarting failed CI job)

**Pull request merged**